pkg/kube/client.go

```diff
@@ -191,7 +191,7 @@ func (c *Client) Update(original, target ResourceList, force bool) (*Result, err
 	for _, info := range original.Difference(target) {
 		c.Log("Deleting %q in %s...", info.Name, info.Namespace)
 		res.Deleted = append(res.Deleted, info)
-		if err := deleteResource(info); err != nil {
+		if err := deleteResource(info); err != nil && !apierrors.IsNotFound(err) {
```

Comment:
I might be missing something but if Helm expects a resource to be present so it tries to delete it but the resource isn't present so an error is thrown... should that be noted?

Reply:
If you are referring to logging it, I think so. I'll make the modification that it will always log the error, but only return an error if it isn't a not found error

Reply:
Does my update address what you were looking for?